src/olympia/scanners/models.py

```diff
@@ -88,6 +90,17 @@ def get_scanner_name(self):
     def get_pretty_results(self):
         return json.dumps(self.results, indent=2)
 
+    def can_report_feedback(self):
+        return (
+            self.has_matches and self.state == UNKNOWN and self.scanner != WAT
+        )
+
+    def get_git_repository(self):
+        return {
+            CUSTOMS: settings.CUSTOMS_GIT_REPOSITORY,
```

Comment:
I don't think we need it yet, but worth keeping in mind: at some point we may want to start defining a class for each scanner with methods like `run()`, `get_git_repository()`...